src/vrp_params.h

```diff
@@ -25,7 +25,7 @@ struct VRPSearchParams {
   std::int32_t numVehicles;
   std::int32_t depotNode;
   std::int32_t timeHorizon;
-  std::int32_t vehicleCapacity;
+  std::vector<int64> vehicleCapacity;   // type changed to vector 
```

Comment:
why do you need int64 ? why not simply std::int32_t. anyways use int64_t if you have to after explaining why you need 64bit integers.